docs/source/model_design/risk_factors.rst

```diff
@@ -1,55 +1,299 @@
+  general document about designing Vivarium models?
+
+  On the other hand, it looks like the following sections in :ref:`Modeling
+  Causes <models_cause>` have details that are specific to causes, but should
+  have analogues for risks in this document:
+
+  * How is a cause model incorporated into a larger model?
+  * What does a model document look like?
+
+.. contents::
+  :local:
+
+Overview
+--------
+
+What is a risk factor?
+++++++++++++++++++++++
+
+What is a causal relationship?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+What is a risk exposure?
+++++++++++++++++++++++++
+
+What is a risk effect?
+++++++++++++++++++++++
+
+Definitions
+-----------
+
+Theoretical Minimum Risk Exposure Level/Distribution (TMREL/D)
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+Measures of occurence
++++++++++++++++++++++
+
+This is a recap: Epidemiology is the study of the distribution and determinants of disease frequency in human populations. Simply put, it is the study of the *occurence* of illness. Measures of disease frequency are tools to describe how common an illness is (or outcome of an event) with reference to the size of the population at risk. They are used to count cases, in relation to a population and to a measure of time. Outcomes can be infection, disease, disability, death, other risk-exposures, recovery or usage of health care. 
+
+There are two main measures of disease occurence/frequency: **prevalence** and **incidence**. Incidence quantifies the occurence of new cases of disease whereas prevalence, a measure of status rather than newly occuring disease, quantifies existing cases. New cases are called **incident cases** and existing cases are called **prevalent cases**. 
+
+.. todo:: 
+  link to the other doc on incidence and prevalence?
+
+Example: 
+
+  * Measure of incidence: 124.2 out of 100,000 women developed breast cancer in the USA in 2016.
+  * Measure of incidence: A study of 3000 children in selected rural areas of Ethiopia looked at the levels of disease and death caused by diarrhoea. It found 4 deaths of diarrhoea per 1,000 children per year. The same study found 360 episodes of diarrhoea per 100 children per year.
+  * Measure of prevalence: 20.7% of women attending antenatal care at rural clinics Siaya county, western Kenya were HIV positive in 2015
+
+Prevalence
+^^^^^^^^^^
+Prevalence focuses on existing states. Prevalence of a state (such as the 'with condition state') at a point in time may be defined as the proportion of a population in that state at that time; thus prevalence is the proportion of persons in a defined population that have the outcome under study in a defined period of time. Prevalence is a proportion and has no time units. However, the point in time to which it refers must always be specified. The term ‘prevalence rate’ is often wrongly used instead of ‘prevalence’. Prevalence is, by definition, a proportion not a rate. 
+

```

Comment:
Maybe we could link out to our descriptions of incidence and prevalence in the modeling causes page? https://vivarium-research.readthedocs.io/en/latest/model_design/cause.html#prevalence